packages/wrangler/src/index.tsx

```diff
@@ -1256,7 +1256,7 @@ export async function main(argv: string[]): Promise<void> {
         );
       }
 
-      const accountId = await requireAuth(config);
```

Comment:
@threepointone Is it an issue that during `--dry-run`, accountId won't be set even if you're logged in? We're not like generating a manifest with accountIds/paths that rely on accountId being set?

Reply:
Well we want zero requests during dry run, so maybe it's good that we enforce it doubly here. We'll want to generate a manifest, but we probably won't include account id in it (yet).